### DISCUSSION TITLE
Using sentiment-free emojis for answers?

### DISCUSSION THREAD

**Discussion by LeaVerou** [Devographics/surveys#188] [General Survey Design]
Currently, there are two very widespread types of questions:

Feature questions
<img width="815" alt="image" src="https://github.com/Devographics/surveys/assets/175836/712132a3-3707-47d0-add9-674ec67d20e0">

Tooling questions
<img width="818" alt="image" src="https://github.com/Devographics/surveys/assets/175836/86df1633-a6bd-4888-ad59-967706c6c17e">

It has been raised a few times that some of these emojis can be misleading, especially in Feature questions, where using ✅ or 👍 can be construed as expressing sentiment, rather than experience/awareness.
There is value in having emojis, as they help visual anchoring, so I wouldn't suggest removing them, but maybe replacing some of the more misleading ones?
For example, "Heard of it" could use 👂, and "Used it" could use ⌨️. These are just ideas, the main point of this is to start the discussion.

The emojis used in the 5-answer questions are less misleading, as they *do* also communicate sentiment, though there's no obvious reason why the 👍👎 pair is used for "Used it" and the ✅🚫 pair for "Heard of it".

**Comment by SachaG** [comment]
As you said the idea behind the emojis is just to anchor the answers visually, so there isn't that much meaning behind them. We could probably replace them with images or color-coding if we think of something clearer.

**Comment by LeaVerou** [comment]
Another idea: 🤷👀🤓

<img width="244" alt="image" src="https://github.com/Devographics/surveys/assets/175836/db34c2ee-583d-40a1-8744-d8ac39d7eaf6">

I was a bit worried that using an ear for "Heard it" may be seen as ableist, so this avoids that. Not sure about 🤓 for used it. Maybe 🧑‍💻? Though 🤓 is more in tune with the lighthearted character of the survey.

**Reply by SachaG** [reply]
I think that especially since we're considering using a 5-option format for features as well as libraries, we should probably try and bring both question formats closer together? So maybe avoid introducing different emojis?

**Reply by SachaG** [reply]
Although I do like the eyes for "heard of it" :)

**Reply by LeaVerou** [reply]
> I think that especially since we're considering using a 5-option format for features as well as libraries, we should probably try and bring both question formats closer together? So maybe avoid introducing different emojis?

This is orthogonal, since the 5-answer format doesn't use the same emojis as the 3-answer format.

**Reply by SachaG** [reply]
My point was that it we are going to change things, we should think about a set of symbols that work for both question templates. The eyes emoji is fun, but we'd ideally need something that can also work for "heard of it > would like to learn" and "heard of it > not interested". So some kind of symbol that can have neutral, positive, and negative versions would be ideal.

**Reply by LeaVerou** [reply]
Oh, I see what you mean now. That would be nice but I don't think that exists (unless we use something generic, like colored shapes — though then you need to localize them to different colors in different locales).

I think for the 5-point answers, it may better to use the same emojis as the 3 point answers, with a "subscript" sentiment emoji (e.g. thumbs up or down), rather than trying to find unicorn emojis that have neutral, positive and negative versions. Though it may need work to be clear at this font size.
